app/main.py

```diff
@@ -1,3 +1,79 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    @staticmethod
+    def check_value(value: [Distance, int]) -> int:
```

Comment:
The `check_value` method should handle both `Distance` and `int` types properly. The current implementation assumes that if `value` has a `real` attribute, it should be used, otherwise it uses `value.km`. This logic might not work correctly if `value` is an `int`. Consider checking the type of `value` explicitly.